### PR TITLE
EventGrid fixes Communication event types

### LINKED ISSUE #19925: AcsChatParticipantAddedToThreadEvent/AcsChatParticipantRemovedFromThreadEvent system event mapping is broken resulting in Microsoft.Communication.ChatThreadParticipantAdded/Microsoft.Communication.ChatThreadParticipantRemoved events not being usable in the SDK
**Describe the bug**
AcsChatParticipantAddedToThreadEvent/AcsChatParticipantRemovedFromThreadEvent system event mapping is broken resulting in Microsoft.Communication.ChatThreadParticipantAdded/Microsoft.Communication.ChatThreadParticipantRemoved events not being usable in the SDK

***Exception or Stack Trace***
Add the exception log and stack trace if available

**To Reproduce**
1. Attempt to parse Microsoft.Communication.ChatThreadParticipantAdded/Microsoft.Communication.ChatThreadParticipantRemoved events using the SDK
***Code Snippet***
Add the code snippet that causes the issue.

**Expected behavior**
Successfully able retrieve event payload Microsoft.Communication.ChatThreadParticipantAdded/Microsoft.Communication.ChatThreadParticipantRemoved 

**Actual behavior (include Exception or Stack Trace)**
What is the actual behavior?
SDK rejects the payload due to missing system event mapping

**Screenshots**

**Setup (please complete the following information):**
 - IDE : [e.g. IntelliJ]
 - Version of the Library used

**Additional context**

**Information Checklist**
Kindly make sure that you have added all the following information above and checkoff the required fields otherwise we will treat the issuer as an incomplete report
- [ ] Bug Description Added
- [ ] Repro Steps Added
- [ ] Setup information Added

### PR DESCRIPTION
**Pull request by YijunXieMS**
The event grid swagger file changed event types in PR https://github.com/Azure/azure-rest-api-specs/pull/13485/files
This PR is to update the SystemEventNames mapping for the swagger change.

Closes #19925

### CODE REVIEW

File: eng/versioning/version_client.txt [hunk 1]
```diff
@@ -78,7 +78,7 @@ com.azure:azure-e2e;1.0.0-beta.1;1.0.0-beta.1
 com.azure:azure-identity;1.2.4;1.3.0-beta.3
 com.azure:azure-identity-perf;1.0.0-beta.1;1.0.0-beta.1
 com.azure:azure-iot-deviceupdate;1.0.0-beta.1;1.0.0-beta.2
-com.azure:azure-messaging-eventgrid;4.0.0;4.1.0-beta.1
+com.azure:azure-messaging-eventgrid;4.0.0;4.1.0
```
Comment (alzimmermsft) [inline eng/versioning/version_client.txt:81]: Out of band release?

**Review by alzimmermsft — APPROVED** [review]

File: eng/versioning/version_client.txt (hunk 1 above)
Comment (YijunXieMS) [reply eng/versioning/version_client.txt:81]: Yes, communication service team changed the communication service related part of the EventGrid swagger file. New system events are added and a mapping error is fixed.
The communication service will GA on 03/29. We need to GA the updates before service GA.

File: sdk/eventgrid/azure-messaging-eventgrid/src/main/java/com/azure/messaging/eventgrid/systemevents/AcsRecordingFileStatusUpdatedEventData.java [hunk 2]
```diff
@@ -77,22 +79,22 @@ public AcsRecordingFileStatusUpdatedEventData setRecordingStartTime(OffsetDateTi
...
+     * Set the recordingDuration property: The recording duration.
      *
-     * @param recordingDurationMs the recordingDurationMs value to set.
+     * @param recordingDuration the recordingDuration value to set.
      * @return the AcsRecordingFileStatusUpdatedEventData object itself.
      */
-    public AcsRecordingFileStatusUpdatedEventData setRecordingDurationMs(Long recordingDurationMs) {
-        this.recordingDurationMs = recordingDurationMs;
+    public AcsRecordingFileStatusUpdatedEventData setRecordingDuration(Duration recordingDuration) {
+        this.recordingDurationMs = recordingDuration.toMillis();
```
Comment (srnagar) [inline sdk/eventgrid/azure-messaging-eventgrid/src/main/java/com/azure/messaging/eventgrid/systemevents/AcsRecordingFileStatusUpdatedEventData.java:97]: We should null check this before calling `toMillis()`.

File: sdk/eventgrid/azure-messaging-eventgrid/src/main/java/com/azure/messaging/eventgrid/systemevents/AcsRecordingFileStatusUpdatedEventData.java [hunk 3]
```diff
@@ -77,22 +79,22 @@ public AcsRecordingFileStatusUpdatedEventData setRecordingStartTime(OffsetDateTi
...
-     * Get the recordingDurationMs property: The recording duration in milliseconds.
+     * Get the {@link Duration} of recordingDuration property: The recording duration.
      *
-     * @return the recordingDurationMs value.
+     * @return the recordingDuration value.
      */
-    public Long getRecordingDurationMs() {
-        return this.recordingDurationMs;
+    public Duration getRecordingDuration() {
+        return Duration.ofMillis(this.recordingDurationMs);
```
Comment (srnagar) [inline sdk/eventgrid/azure-messaging-eventgrid/src/main/java/com/azure/messaging/eventgrid/systemevents/AcsRecordingFileStatusUpdatedEventData.java:87]: Do a null check.

File: sdk/eventgrid/azure-messaging-eventgrid/CHANGELOG.md [hunk 4]
```diff
@@ -1,7 +1,13 @@
...
 
-## 4.1.0-beta.1 (Unreleased)
+## 4.1.0 (2020-03-24)
+### New Features
+- Added new system event model class `AcsRecordingFileStatusUpdatedEventData`.
+- Added new attribute `transactionId` to `AcsChatEventInThreadBaseProperties`.
+- Added new attribute `tag` to `AcsSmsDeliveryReportReceivedEventData`.
 
+### Bug Fixes
+- Fixed event types in `SystemEventNames` for system event data `AcsChatParticipantRemovedFromThreadEventData` and `AcsChatParticipantAddedToThreadEventData`.
```
Comment (srnagar) [inline sdk/eventgrid/azure-messaging-eventgrid/CHANGELOG.md:10]: Include the new constant added and the deprecated constants.

**Review by srnagar — APPROVED** [review]

**Pull request merged**